app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointmentRow.kt

```diff
@@ -103,6 +105,14 @@ data class OverdueAppointmentRow(
 
     holder.patientNameTextView.text = context.getString(R.string.overdue_list_item_name_age, name, age.toString())
     holder.patientNameTextView.setCompoundDrawableStart(gender.displayIconRes)
+    holder.patientAddressTextView.text = when {
+      patientAddress.streetAddress != null && patientAddress.colonyOrVillage != null -> {
+        "${patientAddress.streetAddress}, ${patientAddress.colonyOrVillage}"
+      }
+      patientAddress.streetAddress != null -> patientAddress.streetAddress
+      patientAddress.colonyOrVillage != null -> patientAddress.colonyOrVillage
+      else -> "${patientAddress.district}, ${patientAddress.state}"
+    }
```

Comment:
One last thing, instead of `null` checks, should we use `!isNullOrBlank` instead?